modules/oe_showcase_news/config/install/field.field.node.oe_news_simple.field_news_publication_date.yml

```
@@ -0,0 +1,23 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_news_publication_date
+    - node.type.oe_news_simple
+  module:
+    - datetime
+id: node.oe_news_simple.field_news_publication_date
+field_name: field_news_publication_date
+entity_type: node
+bundle: oe_news_simple
+label: 'Publication date'
+description: ''
+required: true
```

it makes no sense to have both default value and required configuration
```suggestion
required: false
```